### DISCUSSION TITLE
Kimai cloud account deleted while I was on summer Holiday.

### DISCUSSION THREAD

**Discussion by stgpepper** [kimai/kimai#4195] [Questions & Answers] [answered]
I tried to log into my Kimai account and noticed it does not go to the usual page I have bookmarked. I found that the registration did not accept my credentials. I asked for password recovery and while waiting message I notice I had received email describing that my account can be permanently deleted since it is unused. Email was sent in 19.6.2023 so just over a month ago. Since I did not receive any email to reset my password I am leaded to believe that my account has already been deleted. 

I've been using Kimai cloud since 2021. I did not host it. It was hosted on a platform that was linked in the Kimai account registration at the time. In the past year I used it daily for my uni studies until uni was done for the year. So I think I last logged into Kimai account in mid of may. 

Is there anything that can be done to recover and to continue using the service?

**Comment by kevinpapst** [comment, marked as answer]
I reply with one link: https://www.kimai.cloud/help/data-removal

But please understand that this repo is about Kimai core, not about the hosted cloud version.
